core/issues/issue_types.py

```diff
@@ -36,6 +36,10 @@ class CephDaemonWarning(IssueTypeBase):
     pass
 
 
+class CephDaemonError(IssueTypeBase):
```

Comment:
I'd like to change this CephDaemonVersionsError since CephDaemonError feels more appropriate for when an error has occured with a specific daemon.

Reply:
ack